node/actors/logger.go

```diff
@@ -26,7 +26,20 @@ type NodeLogger struct {
 func (nl *NodeLogger) HandleLog(message gen.MessageLog) error {
 	// TODO: this is temporary until we can figure out what we really want
 	// to do with these logs
-	nl.inner.Log(context.Background(), slog.Level(message.Level), fmt.Sprintf(message.Format, message.Args...))
+	switch message.Level {
```

Comment:
why is this switch statement necessary when the call to `Log` did the job?

Reply:
The log message doesn't do the job.  Here's why ->
From ergo:
```
const (
	LogLevelSystem LogLevel = -100

	LogLevelTrace    LogLevel = -2
	LogLevelDebug    LogLevel = -1
	LogLevelDefault  LogLevel = 0 // inherite from node/app/parent process
	LogLevelInfo     LogLevel = 1
	LogLevelWarning  LogLevel = 2
	LogLevelError    LogLevel = 3
	LogLevelPanic    LogLevel = 4
	LogLevelDisabled LogLevel = 5
)
```
From slog: [link](https://pkg.go.dev/log/slog#Level)
```
const (
	LevelDebug Level = -4
	LevelInfo  Level = 0
	LevelWarn  Level = 4
	LevelError Level = 8
)
```

As you can see, the mappings are incorrect.  The ergo author either rolled his own or used something not IAW the go standard lib

Reply:
Logs without the translation: 
```
[INFO] 2024-09-30 14:05:49 - Validating Nex Node
[INFO] 2024-09-30 14:05:49 - Starting Nex Node
[] 2024-09-30 14:05:49 - Nex node started
[] 2024-09-30 14:05:49 - Observer Application started and available at http://localhost:9911/
[] 2024-09-30 14:05:49 - registered publishable event 'internal_nats_server_ready', waiting for consumers...
[] 2024-09-30 14:05:49 - successfully subscribed to: Event#<EC87E8E8:'internal_nats_server_ready'>
[] 2024-09-30 14:05:49 - publisher got first consumer for 'internal_nats_server_ready'. start producing events...
[] 2024-09-30 14:05:49 - received event Event#<EC87E8E8:'internal_nats_server_ready'>
[] 2024-09-30 14:05:49 - starting 0 agent binaries..
``` 
(missing level information 

Logs after fix in place 
```
[INFO] 2024-09-30 14:26:46 - Validating Nex Node
[INFO] 2024-09-30 14:26:46 - Starting Nex Node key=derp
[INFO] 2024-09-30 14:26:46 - Nex node started%!(EXTRA slog.Attr=sadf=Asdf)
[INFO] 2024-09-30 14:26:46 - Observer Application started and available at http://localhost:9911/
[INFO] 2024-09-30 14:26:46 - registered publishable event 'internal_nats_server_ready', waiting for consumers...
[INFO] 2024-09-30 14:26:46 - successfully subscribed to: Event#<EC87E8E8:'internal_nats_server_ready'>
[INFO] 2024-09-30 14:26:46 - publisher got first consumer for 'internal_nats_server_ready'. start producing events...
[INFO] 2024-09-30 14:26:46 - received event Event#<EC87E8E8:'internal_nats_server_ready'>
[INFO] 2024-09-30 14:26:46 - starting 0 agent binaries..
```

Reply:
Thanks @jordan-rash.

Marking this as resolved. @autodidaddict please unresolve if your concern wasn't addressed.